guides/common/modules/ref_system-requirements.adoc

```diff
@@ -94,6 +94,16 @@ ifdef::foreman-el,katello,satellite[]
 SELinux must be enabled, either in enforcing or permissive mode.
 Installation with disabled SELinux is not supported.
 
+.Synchronized system clock
```

Comment:
this is inside a `ifdef::foreman-el,katello,satellite[]` block, but it applies to *alll* deployments (so also Debian and Orcharino)

Reply:
Addressed.